_posts/2019-08-02-Rates-Rebates-Recommendations-Report.md

```diff
@@ -0,0 +1,276 @@
+1. The second trial should be limited to the same three councils who participated in June/July. This will enable valuable data to be gathered while keeping the cost of the extension minimal.
+1. Delay the decision about whether to proceed beyond this extension until the end of this second trial. This will allow time for results from the first round to be validated against higher application volumes.
+1. Improvements should be made to how data is transferred between the online application solution and council core systems, and options explored to improve data transfer between council systems and eRMS. The high trust relationship built with the councils involved in the trial should be leveraged to (co)design and implement simple, low cost (but high value) solutions during the trial extension period
+
+## Options to consider
+
+- Option 1 - **Close down the initiative**: inform Councils there will be no change to the current paper-based service from DIA. Decommission the service.
+- Option 2 - **Run a second trial with the existing three councils**. Continue to provide support, but limit other activity to keep costs minimal.
+- Option 3 - **Run a second trial with the existing three councils through a high application volume period and implement data export solutions (recommended)**: proceed as for option two, but also implement data export solution for councils (for import into their core systems). Investigate (and document a recommendation for) the best solution to improve data submission (reduce cost and reduce/eliminate data entry errors) from council core systems to eRMS.
+- Option 4 - **Extend the trial with the existing three councils until the end of June 2020**: continue to gradually evolve the service with these councils throughout this period. Use the high quality relationship built with councils to co-design the optimal online (and paper) application process ready for when the statutory declaration requirement is removed.
+- Option 5 - **Extend the trial until the end of June 2020 and allow another three councils to join**: continue to run the trial with Tauranga City Council, Kāpiti District Council, and Hutt City Council through to June 2020 and trial the service with an additional three councils. Aim to sign up councils that will best test whether the service is ready to be rolled out nationally.
+
+## Background
+
+The Rates Rebates Online Beta was designed to meet the needs of ratepayers, Councils, and the DIA Rates Rebates team. It was built based on the findings of a Discovery and Alpha phase.
+
+In the Discovery phase the Rates Rebates service was mapped and reviewed, internal and external stakeholders were interviewed, and potential methods of service delivery were explored. The discovery revealed many opportunities at the system level to improve the experience for both the people applying for their entitlement and people delivering the service.
```

Comment:
"the Rates Rebates service was mapped and reviewed"
Can be made active by saying who mapped the service.

Reply:
```suggestion
In the Discovery phase we mapped and reviewed the existing Rates Rebates delivery system. We interviewed internal and external stakeholders, and explored potential methods of service delivery. The discovery revealed many opportunities at the system level to improve the experience for both the people applying for their entitlement and people delivering the service.
```